bitnami/apache/templates/prometheusrules.yaml

```diff
@@ -13,7 +13,9 @@ metadata:
     {{- if .Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
+    {{- if .Values.metrics.prometheusRule.additionalLabels }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.prometheusRule.additionalLabels "context" $ ) | nindent 4 }}
+    {{- end }}
```

Comment:
Since it is already broken I think it is not being used, I mean, the current code is not usable so we don't need to care about compatibility since it is not working, I would opt for directly use the new standard and not complicate the logic